jans-cedarling/bindings/cedarling_python/PYTHON_TYPES.md

```diff
@@ -113,12 +113,10 @@ authorization data with access token, action, resource, and context.
 
 Attributes
 ----------  
+:param tokens: Python dictionary with additional context.  
```

Comment:
Is this a python dictionary? From `cedarling_python.pyi` line 143 I see Tokens is a class.

Reply:
it's a class. let me update the docs again

Reply:
should be fixed now